### ISSUE TITLE
Document styles not applied in DataTemplates

### ISSUE THREAD

**Issue by klasyc** [whistyun/MdXaml#18]
Hello,

I am trying to use `MarkDownScrollViewer` inside an `<ItemsControl>` element. The markdown content is loading properly, but the provided style name is ignored and the text is rendered always with some default style. I also tried to pass a new style from the ResourceDictionary, but the result is just the same. 

Maybe this behavior could be related to [this](https://stackoverflow.com/questions/2476305/wpf-some-styles-not-applied-on-datatemplate-controls)?

```
...
<!-- This prints "Hello world" correctly in "compact" style -->
<mdxam:MarkdownScrollViewer xml:space="preserve" Margin="0 10 0 10" ScrollViewer.VerticalScrollBarVisibility="Auto" MarkdownStyleName="Compact">
# Hello world
</mdxam:MarkdownScrollViewer>
...
<ItemsControl ItemsSource="{Binding MyAwesomeCollection}">
    <ItemsControl.ItemTemplate>
        <DataTemplate>

            <!-- But this prints "Hello world" in default style -->
            <mdxam:MarkdownScrollViewer xml:space="preserve" Margin="0 10 0 10" ScrollViewer.VerticalScrollBarVisibility="Auto" MarkdownStyleName="Compact">
            # Hello world
            </mdxam:MarkdownScrollViewer>

        </DataTemplate>
    </ItemsControl.ItemTemplate>
</ItemsControl>
```

**Comment by klasyc** [comment]
Thank you for the fix, tried current develop branch and it works correctly. Do yo plan to release the fix in Nuget package?

**Comment by whistyun** [comment]
Thank you for reporting issue and checking my commit.
I will release it around 4am (UTC) on Feb 11 to change other changes.

**Comment by klasyc** [comment]
Thank you very much. One more thing: it would be helpful to describe the minimal build environment in README to make these contributions easier. For example my Visual Studio 2017 is not compatible with the project files, so that I had to create a new project and import everything by hand :)

**Comment by klasyc** [comment]
Hello, today I discovered that `MarkdownScrollViewer` ignores `ClickAction` property when defined in `DataTemplate`. Since the behavior and steps to reproduce are the same as for recently fixed `MarkdownStyleName` property, I am "recycling" this issue instead of creating a new one.

**Comment by whistyun** [comment]
I released v1.11.0 to notice ‘ClickAction` changing in DataTemplate.

**Comment by klasyc** [comment]
It works. Thank you again!
